pkg/network-engine/gateway.go

```diff
@@ -62,8 +63,13 @@ func (l *libreswanGateway) Start() {
 		klog.Errorf("fail to create secrets file: %v", err)
 	}
 	defer file.Close()
-	// TODO: secret PSK config
-	fmt.Fprintf(file, "%%any %%any : PSK \"%s\"\n", "alibaba")
+
+	psk := os.Getenv("PSK")
```

Comment:
PSK is libreswan’s config.   Maybe  `LIBRESWAN_PSK` or `BACKEND_LIBRESWAN_PSK` as env name will be better?   

@DrmagicE Any suggestion?

Reply:
done

Reply:
LIBRESWAN_PSK +1